xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -44,7 +44,7 @@
 #define ADDON_GLOBAL_VERSION_GENERAL_XML_ID           "kodi.binary.global.general"
 #define ADDON_GLOBAL_VERSION_GENERAL_DEPENDS          "General.h"
 
-#define ADDON_GLOBAL_VERSION_GUI                      "5.12.0"
+#define ADDON_GLOBAL_VERSION_GUI                      "5.12.1"
```

Comment:
Is this a compatible API change, means, addons must not be recompiled?

Reply:
Yes, it add only the header for time and for the DirectX added function is on a by `malloc` created structure entry on end.  This is then a bit bigger but becomes never edited on addon side.

https://github.com/xbmc/xbmc/pull/15609/files/a51776e52d7301a7de8e39a28b1b7efdffd7b6c1#diff-c244971ddb8feeb77e6e8a8c7af39a99R29

Reply:
Key for compatibility is to add the new function table entry at the end of the table, as you did. All fine, then.